src/components/FlowStep/FlowStep.stories.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react"
+import { Meta, Story } from "@storybook/react"
+import { Container, Link, Image } from "@chakra-ui/react"
+import customDescriptionImage from "./customDescriptionImage.png"
```

Comment:
I don't like the name of the file tbh because it doesn not say anything specific about the image. It is a graphic illustration of how deposit bitcoin address is created do maybe `DepositBTCAddresCreationImage`? 🤔 Maybe a little too long so any ideas of a better name are welcome 😅

Reply:
I think `DepositBTCAddresCreationImage` is fine. I guess I chose `customDescriptionImage` because the idea is that it could be anything. But I agree - we should name the file with a better description of what is *is*, not what it's used for.